addons/payment_adyen/views/payment_adyen_templates.xml

```diff
@@ -3,6 +3,7 @@
 
     <template id="inline_form">
         <div t-attf-id="o_adyen_dropin_container_{{acquirer_id}}" class="o_adyen_dropin"/>
+        <div id="o_adyen_paypal_component" class="d-none"/>
```

Comment:
We cannot update the template in stable because it is saved as a view record, which only updates its `arch` when the server is started with `-u payment_adyen`.

You can, however, create and destroy a `div` on-the-fly in JS.